guard-falcon.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
 	spec.add_dependency("guard")
 	spec.add_dependency("guard-compat", "~> 1.2")
 	
-	spec.add_development_dependency "bundler", "~> 1.3"
+	spec.add_development_dependency "bundler", "~> 2.0"
```

Comment:
For this to work correctly, just remove the version specification. Otherwise travis blows up occasionally.

`spec.add_development_dependency "bundler"`